src/fb.md

```diff
@@ -246,13 +243,18 @@ after the first stage.
   TTk.AlgorithmPuzzle(3)
     .size({width:400, height:400})
     .fc('wttwttwtttttttttttrttrttttttttttttttbbbbbbtttttottottt')
-    .case("B' L' B2 D2")
+    .case("R F' D R' U' B")
     ('#inf13');
 </script>
 
-`L' B2` builds the bottom line of FB and orients the blue-red edge.
+`F' D` builds the front square. 
+Back pair's corner is clearly uninfluencable, so predict the back pair's corner's position and orientation, and then aim to influence the back pair's edge. 
 
-Starting with `B'` leverages the `B2` to also insert the back edge, for a `line + line` block.
+We can predict the back pair's corner will be at DBR after first stage, with its D-face sticker on the bottom. 
+
+We should know from experience that the back pair's edge needs to be misoriented in BR to pair with the back pair's corner, when the back pair's corner is in DBR with its D-face sticker on the bottom.
```

Comment:
remove the comma?

Reply:
Yes, thanks. Was worried about sentence getting too long, but it's better now.